tests/integration/minibroker_test.go

```diff
@@ -58,10 +71,65 @@ var _ = Describe("classes", func() {
 		assert func(*apiv1beta1.ServiceInstance, *apiv1beta1.ServiceBinding)
 	}{
 		{
-			name:   "mariadb",
-			plan:   "10-3-22",
-			params: map[string]interface{}{},
-			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {},
+			name: "mariadb",
+			plan: "10-3-22",
+			params: map[string]interface{}{
+				"db": map[string]interface{}{
+					"name": "mydb",
+					"user": "admin",
+				},
+			},
+			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {
+				By("rendering and loading the mariadb client template")
+				tmplPath := path.Join(testDir, "resources", "mariadb_client.tmpl.yaml")
+				values := map[string]interface{}{
+					"DatabaseVersion": "10.3",
+					"SecretName":      binding.Spec.SecretName,
+					"Command": []string{
+						"sh", "-c",
+						"mysql" +
+							" --host=${DATABASE_HOST}" +
```

Comment:
I think if we called this [`MYSQL_HOST`](https://dev.mysql.com/doc/refman/8.0/en/environment-variables.html) instead we don't need it as a command line argument. Also `MYSQL_PWD`. Not sure there's anything for user and DB name, though, so I guess none of this matters.

Reply:
Yeah, we don't have a comprehensive mapping for all the arguments. I'd rather keep it this way for consistency and clarity.